app/components/edit-text/style.scss

```diff
@@ -1,12 +1,42 @@
-.edit-text-title{
-	border: none;
+
+
+.ql-editor{
+    white-space: inherit;
+    padding:0;
+} 
+
+.left {
+    text-align: left;
+    border: none;
+    background: rgba(158, 158, 158, 0.08);
+    font-size: 30px;
+    width:100%;
+    border-radius: 2px;
+}
+.center {
+    text-align: center;
+    border: none;
```

Comment:
But semantically speaking, these are not related at all to alignment, maybe you need a different class there